crates/astria-sequencer/src/main.rs

```diff
@@ -24,6 +24,13 @@ async fn main() -> ExitCode {
         config = serde_json::to_string(&config).expect("serializing to a string cannot fail"),
         "initializing sequencer"
     );
+
+    #[cfg(feature = "mint")]
+    {
+        tracing::warn!("MINT FEATURE IS ENABLED!");
```

Comment:
Can you spin up a task that prints this every 5 seconds?

Also, it should also be written to stderr (`eprintln!`)

Reply:
why stderr if this isn't an error?

Reply:
updated though!

Reply:
stderr is not only used for errors but also other information like debug, config, etc. stdout is the intended behaviour of a program (think counting the lines of a program) and stderr for a summary of input flags, files, debug info etc

Reply:
updated